templates/default/kitchen.cloud.yml.erb

```diff
@@ -23,11 +26,11 @@ provisioner:
       disabled: true
 
 platforms:
-- name: centos-6.5
+- name: centos-6.6
   driver_plugin: openstack
   driver_config:
     username: centos
-    image_ref: "CentOS 6.5"
+    image_ref: "CentOS 6.6"
 - name: ubuntu-12.04
```

Comment:
I don't believe we run any Ubuntu, so this change makes a lot of sense. For cookbook we would like to open source and support multiple platforms we can manually add them in.